jwst/resample/resample_step.py

```diff
@@ -116,8 +118,20 @@ def process(self, input_data):
             result = resamp.resample_many_to_many(in_memory=self.in_memory)
 
         else:
+            if self.enable_var:
+                # If variance is enabled, we compute the error from the variance
+                compute_err = "from_var"
+            else:
+                # compute_err = "from_var" causes enable_var to be ignored, so must set it to the
+                # other option even though variance is not computed at all.
+                compute_err = "driz_err"
```

Comment:
```suggestion
                compute_err = None
```
Looking at stcal I think `None` is supported here. Setting this to `driz_err` looks to trigger [extra drizzling](https://github.com/spacetelescope/stcal/blob/8b5e79653ba044701e01fbf8d1ea59fd1ac3b60d/src/stcal/resample/resample.py#L624) (which I think setting to `None` avoids).

Reply:
> ok, thanks, seems confusing that this still does anything when compute_var is turned off

Errors can be computed either by resampling input error array or by resampling variance array. So turning on/off resampling variance array should not control whether error array is computed.

Reply:
The driz_err option should only be used during outlier detection -- for that use case, it needs an interpolated error array to median-combine, not a propagated variance.  I think Brett is right here that if the user turns off variance computation, no error computations should occur.